src/gluonts/time_feature/lag.py

```diff
@@ -85,9 +85,14 @@ def _make_lags_for_month(multiple, num_cycles=3):
     # multiple, granularity = get_granularity(freq_str)
     offset = to_offset(freq_str)
 
-    if offset.name == "M":
+    if offset.name == "A-DEC":  # Y
```

Comment:
Can we have make the comment a bit more expressive? I assume these are aliases for pandas frequencies?

Reply:
Yeah its just the normalized frequency string (like we already use for W-SUN)